<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wfSystemWindow</class>
 <widget class="QMainWindow" name="wfSystemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HOnlineMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOnline"/>
   <addaction name="actionOrder"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/printer.png</normaloff>:/images/printer.png</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </action>
  <action name="actionZoomin">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_in.png</normaloff>:/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>zoomin</string>
   </property>
  </action>
  <action name="actionZoomout">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_out.png</normaloff>:/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>zoomout</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
  </action>
  <action name="actionZoomsame">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/zoom_same.png</normaloff>:/images/zoom_same.png</iconset>
   </property>
   <property name="text">
    <string>zoomsame</string>
   </property>
  </action>
  <action name="actionOnline">
   <property name="text">
    <string>online</string>
   </property>
   <property name="toolTip">
    <string>在线浏览</string>
   </property>
  </action>
  <action name="actionOrder">
   <property name="text">
    <string>order</string>
   </property>
   <property name="toolTip">
    <string>模拟开票</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
